lib/crowdnode.js

```diff
@@ -7,7 +7,7 @@ let CrowdNode = module.exports;
 const DUFFS = 100000000;
 
 let Dash = require("./dash.js");
-let Dashcore = require("@dashevo/dashcore-lib");
+let Dashcore = require("@dashevo/dashcore-lib/dist/dashcore-lib.min.js");
```

Comment:
I think this would break node. Will need to test.

Reply:
Check out the latest changes, see if those are acceptable.